velox/exec/ContainerRowSerde.cpp

```diff
@@ -415,31 +408,30 @@ int compare<TypeKind::ROW>(
     auto child = row->childAt(i);
     auto leftNull = bits::isBitSet(nulls.data(), i);
     auto rightNull = child->isNullAt(wrappedIndex);
-    if (leftNull && rightNull) {
-      continue;
-    }
-    if (leftNull) {
-      return flags.nullsFirst ? -1 : 1;
-    }
-    if (rightNull) {
-      return flags.nullsFirst ? 1 : -1;
+
+    if (leftNull || rightNull) {
+      auto result = BaseVector::compareNulls(leftNull, rightNull, flags);
+      if (result && *result == 0) {
```

Comment:
I find this a bit hard to read. Consider being more explicit:

```
if (result.has_value() && result.value() == 0)
```

Reply:
Fixed